gpgs/src/java/com/defold/gpgs/GpgsJNI.java

```diff
@@ -171,6 +170,32 @@ private void sendSimpleMessage(int msg, String key_1, int value_1, String key_2,
         gpgsAddToQueue(msg, message);
     }
 
+    private void sendFailedMessage(int msg, String error_text, Exception e) {
+        if(e != null) {
+
+            if (e instanceof ApiException) {
+                ApiException apiException= (ApiException) e;
```

Comment:
Teeny tiny nitpick: Missing a space before the equals sign :-)

Reply:
Sorry I missed )